napi-inl.h

```diff
@@ -3074,6 +3074,17 @@ inline Value EscapableHandleScope::Escape(napi_value escapee) {
   return Value(_env, result);
 }
 
+////////////////////////////////////////////////////////////////////////////////
```

Comment:
This seems to add just a method as opposed to a class with methods.  I think in earlier discussion it was either going to be a MemoryManagement class (or may Util class) with the method to adjust the memory.

Reply:
It was my misunderstanding. Now I will provide to create a class that expose the method. Sorry.